### DISCUSSION TITLE
bodyParser

### DISCUSSION THREAD

**Discussion by casoetan** [sergiodxa/remix-utils#70] [General]
Why was `bodyParser` removed? What alternatives are available?

**Comment by sergiodxa** [comment]
You can use request.formData() or request.json() instead

**Comment by casoetan** [comment]
Thanks! Any reason why you specifically removed it? It will be worth stating it in the docs and letting others know the alternative. Possibly with an example 🙂

**Reply by sergiodxa** [reply]
The v2 docs marked it as deprecated for a long time, and suggested in JSDoc comments to use Remix built in alternatives

**Reply by casoetan** [reply]
Apologies. I must have somehow missed it.

**Reply by sergiodxa** [reply]
Don't worry, it's not strange to miss that
